synapse/models/infotech.py

```diff
@@ -69,6 +205,17 @@ def getBaseModels():
                     'optfields': 'str=it:dev:str,int=it:dev:int,bytes=file:bytes',
                     'doc': 'A Windows registry key/value pair.'}),
 
+                ('it:semver', {'ctor': 'synapse.models.infotech.SemverType',
+                                'doc': 'Semantic Version type.'}),
+                ('it:prod:soft', {'subof': 'guid', 'doc': 'An arbitrary, unversioned software product.'}),
+                ('it:prod:softver', {'subof': 'guid', 'doc': 'An version of a particular software product.'}),
+                ('it:hostsoft', {'subof': 'comp',
+                                 'fields': 'host,it:host|softver,it:prod:soft:vers',
```

Comment:
the second field here was maybe supposed to be it:prod:softver?

Reply:
oooo good catch.

Reply:
fixed